.github/workflows/i18n-consistency-checker.yaml

```diff
@@ -17,88 +12,70 @@ jobs:
       matrix:
         language: [ja, zh]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: '0'
-      - name: Check consistency
+      - name: Check consistency and create issue
         id: check-consistency
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
Is this token required by the gh cli? `gh issue list`

Was wondering if this is strictly necessary, as I thought that the `GITHUB_TOKEN` that is generated for every workflow run would be available in the env automatically? May be mistaken though but would like to learn how this really works in GHAs :)

Reply:
I just read the document too, and yes, I agree. I think you are very right. I'd like to see how it behaved, but I don't think this is necessary.

> Important: An action can access the GITHUB_TOKEN through the github.token context even if the workflow does not explicitly pass the GITHUB_TOKEN to the action. As a good security practice, you should always make sure that actions only have the minimum access they require by limiting the permissions granted to the GITHUB_TOKEN. For more information, see "[Permissions for the GITHUB_TOKEN](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token)."

https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Reply:
hehe, well the output of [your test](https://github.com/InnerSourceCommons/InnerSourcePatterns/actions/runs/6009925339/job/16300303176) looks pretty clear :)
> gh: To use GitHub CLI in a GitHub Actions workflow, set the GH_TOKEN environment variable. Example:
  env:
    GH_TOKEN: ${{ github.token }}